### PR TITLE
test(snowflake): extend snowflake example dag to run multiple quries in one run

### PR DESCRIPTION
**Pull request by Lee-W**
as title

### CODE REVIEW

File: astronomer/providers/snowflake/example_dags/example_snowflake.py [hunk 1]
```diff
@@ -24,6 +24,12 @@
     "Results in an ASCII table:\n```{{ results_df | tabulate(tablefmt='pretty', headers='keys') }}```"
 )
 
+MULTIPLE_QUERY_IN_ONE_RUN = (
```
Comment (phanikumv) [inline astronomer/providers/snowflake/example_dags/example_snowflake.py:27]: The table name needs to change as "sample_table" is already being created in task above

Comment (Lee-W) [reply astronomer/providers/snowflake/example_dags/example_snowflake.py:27]: That's why I put it as the last task. But sure, we can change it to another name

**Review by phanikumv — APPROVED** [review]

File: astronomer/providers/snowflake/example_dags/example_snowflake.py [hunk 2]
```diff
@@ -66,8 +74,13 @@
         task_id="snowflake_op_sql_select_stmts", sql=SNOWFLAKE_SLACK_SQL, return_last=False
     )
 
+    snowflake_op_multiple_query_in_one_run = SnowflakeOperatorAsync(
```
Comment (pankajkoti) [inline astronomer/providers/snowflake/example_dags/example_snowflake.py:77]: Is the objective of this task something different than the above `snowflake_op_sql_multiple_stmts` task?

Comment (phanikumv) [reply astronomer/providers/snowflake/example_dags/example_snowflake.py:77]: yes we basically want create, insert and drop in the same batch

**Pull request merged**